src/platforms/kotlin-multiplatform/initialization-strategies.mdx

```diff
@@ -62,42 +65,48 @@ func application(
 
 Platform-specific initializers allow for customization of the SDK's configuration options on a per-platform basis. This approach can be particularly useful when your SDK requires platform-specific functionality or performance optimizations.
 
-With platform-specific initializers, you can initialize the SDK directly in the target platform's codebase. This approach gives you the flexibility to fine-tune the SDK's behavior to meet the unique needs of each platform.
+This approach gives you the flexibility to fine-tune the SDK's behavior to meet the unique needs of each platform.
+
+### Common
+
+```kotlin {filename:commonMain/AppSetup.kt}
+import io.sentry.kotlin.multiplatform.Context
+
+expect fun initializeSentry(context: Context? = null)
+```
 
 ### Android
 
-```kotlin {filename:MainActivity.kt}
```

Comment:
By removing the snippet from `YourApplication : Application` (Android) and `func application` (iOS), I wonder how users would know when to call `initializeSentry`.
It's important to know where this should be placed, can we keep this?

Reply:
Yup makes sense